### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by NaelDevn** [Merlode11/pronote-bot-discord#6]
ne veut pas fournir le menu ""TypeError: Cannot read properties of undefined (reading 'map')""

Étapes pour reproduire le problème
1. Aller sur '/'
2. Cliquer sur '/menu'
3. Descendre tout en bas '....'
4. L'erreur est apparue

**Comportement attendu**
Une description claire et concise de ce que vous attendiez.

**Screenshots**
Si c'est possible, joignez des captures d'écran pour que nous comprenions mieux

**Autres informations**
Ajoutez toutes les autres remarques à propos du bogue ici

**Comment by Merlode11** [comment]
La commande menu est très mal développée comme expliqué dans le readme et dans l'issue #4 car je n'ai pas accès avec mon pronote à des menus.
Pour résoudre ces problèmes, serait-il possible d'avoir la réponse à la requête pour avoir les menus ?
Merci !

**Comment by NaelDevn** [comment]
j'ai pas compris

**Comment by Merlode11** [comment]
Tout le message ou seulement une partie ?

**Comment by NaelDevn** [comment]
on peut voc ??

**Comment by Merlode11** [comment]
Oui, bien sûr, tu peux m'envoyer une demande d'ami sur Discord sur `Merlode#8128`

**Comment by Merlode11** [comment]
J'ai corrigé le problème. Normalement il devrait fonctionner mais ce n'est pas encore sûr.
